cockroachcloud/cloud-sso-sql.md

```diff
@@ -0,0 +1,140 @@
+
+???!!! {Cameron, how does this format actually work, does the "@gmail" go in there???}
+
+- [`ccloud`, the {{ site.data.products.db }} CLI](ccloud-get-started.html) must be installed on your local workstation.
+
+
+1. First authenticate to your {{ site.data.products.db }} organization. This command will cause your workstation's default browser to open to a {{ site.data.products.db }} authentication portal. Authenticate here as you normally do to the {{ site.data.products.db }} Console. The `ccloud` utility will receive an authentication token from the browser, allowing you to authenticate from the command line.
+
+	{% include_cached copy-clipboard.html %}
+	~~~shell
+	ccloud auth login
+	~~~
+
+1. You may then use the `ccloud` utility to authenticate to your {{ site.data.products.db }} cluster, allowing you to access the SQL interface.
+
+	{% include_cached copy-clipboard.html %}
+	~~~shell
+	ccloud cluster sql --sso {your cluster name}
+	~~~
+
+## Authenticate software users (service accounts) with external IdPs
+
+Currently, {{ site.data.products.db }} can only serve as a token issuer for human users. Authenticating service accounts, i.e. user identities to be controlled by software applications or scripts, rather by humans, is considerably more complicated, as it requires the user, or more realistically, an IdP admin, to provision the appropriate JWT token.
+
+{{ site.data.products.db }} SSO supports the use of external IdPs such as Google, Microsoft, GitHub, or customer deployed OIDC or SAML solutions, such as Okta. All of these options support multifactor authentication (MFA).
+
+**Learn more:**
+
+This [Cockroach Labs blog post](https://www.cockroachlabs.com/blog/) covers and provides further resources for a variety of auth token-issuing use cases, including using Okta and Google Cloud Platform to issue tokens.
+
+**Prerequisites:**
+
+- You must have the ability to create identities and issue tokens with a SAML or OIDC identity provider.
+	{{site.data.alerts.callout_success}}
+	The `issuer` and `audience` fields in the token must match values [configured in your cluster settings](#configure-your-cluster-to-accept-your-external-identity-provider).
+	{{site.data.alerts.end}}
+- You must have a user identity on a {{ site.data.products.db }} organization. For help setting up an organization and cluster, see: [Quickstart with CockroachDB](quickstart.html).
+- You must have access to a member of the [`admin` role](../{{site.versions["stable"]}}/security-reference/authorization.html#admin-role) in order to update cluster settings, which is required to add an external token issuer/IdP.
```

Comment:
Actually it can be done by just having the new MODIFYCLUSTERSETTING permission as well - https://www.cockroachlabs.com/docs/stable/security-reference/authorization.html#supported-privileges